### PR TITLE
Add registered systems check to VAOS

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This adds a check that requires a user to have an ICN and at least one registered facility. Messages are adapted from legacy VAOS.

## Testing done
Local and unit

## Screenshots
Screen Shot 2019-11-06 at 9 43 21 AM
Screen Shot 2019-11-06 at 9 43 02 AM

## Acceptance criteria
- [ ] Can't view VAOS without a registered system

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: script/run-unit-test.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ if [[ -z "$1" ]] || [[ $1 == -* ]]; then
   exit $?
 fi
 
-BABEL_ENV=test $MOCHA --opts src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js "$@"
+BABEL_ENV=test $MOCHA --opts src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js --recursive "$@"
```
Comment (jbalboni) [inline script/run-unit-test.sh]: https://dsva.slack.com/archives/C5HP4GN3F/p1573056471340700

**Review by narin — APPROVED** [review]
LGTM

**Comment by peggygannon** [comment]
@jbalboni Please see my content suggestions for the messages you mocked up above^^

**We're sorry. We can't find any VA medical facility registrations for you**
To schedule an appointment online or to request Community Care assistance, you need to be:
	Enrolled in VA health care, **and** 
	Registered with at least 1 VA medical center that accepts VA online scheduling

To register with a facility, or if you think this message is an error, contact [your local VA health care facility's](va.gov/find-locations/) registration office. 

-----

**We're sorry. We can't find a record of your VA health care enrollment**
To schedule an appointment online or to request Community Care assistance, you need to be:
	Enrolled in VA health care, **and** 
	Registered with at least 1 VA medical center that accepts VA online scheduling

[Find out how to apply for VA health care benefits](va.gov/health-care/how-to-apply/)

**Comment by jbalboni** [comment]
Thanks @peggygannon!

**Review by rianfowler — APPROVED** [review]

**Pull request merged**